lib/fog/ovirt/compute.rb

```diff
@@ -140,18 +160,18 @@ def initialize(options={})
           connection_opts[:ca_no_verify]  = options[:ovirt_ca_no_verify]
           connection_opts[:filtered_api]  = options[:ovirt_filtered_api]
 
-          @client = OVIRT::Client.new(username, password, url, connection_opts)
+          @client = ClientWrapper.new(OVIRT::Client.new(username, password, url, connection_opts))
         end
 
         def api_version
           client.api_version
+        rescue ::OVIRT::OvirtException => e
```

Comment:
No need for this, we already do it in the `client`

Reply:
opps, missed that point, sorry